vra/resource_machine.go

```diff
@@ -24,38 +25,37 @@ func resourceMachine() *schema.Resource {
 		Delete: resourceMachineDelete,
 
 		Schema: map[string]*schema.Schema{
-			"flavor": &schema.Schema{
```

Comment:
Oh good, looks like this got alphabetized. Cool!

Reply:
Yes.